core/unit_test/cuda/TestCuda_InterOp_Graph.cpp

```diff
@@ -97,10 +97,10 @@ TEST_F(TEST_CATEGORY_FIXTURE(GraphInterOp), count_nodes) {
 TEST_F(TEST_CATEGORY_FIXTURE(GraphInterOp), debug_dot_print) {
 #if CUDA_VERSION < 11600
```

Comment:
How is that one fine but not the ones below?

Reply:
It is a `Cuda` interoperability test, so I guess compiling this test without `CUDA_VERSION` could not happen ?